__tests__/thesis-editor-test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
This is a jest snapshot test. Not much going on just yet, but it does some basic smoke testing.